pipelines/_ingredients/MetricsMiscCalibrateImage.yaml

```diff
@@ -0,0 +1,46 @@
+# TODO DM-40388: This file will replace MetricsMisc.yaml, once the new pipeline is vetted.
+# Miscellaneous metrics for Alert Production
+# In the future, these might be placed in task-specific pipelines (for debugging)
+# or grouped by their datasets (to optimize expensive Butler reads)
+
+description: Miscelaneous AP Pipeline metrics
+tasks:
+  numNewDiaObjects:
+    class: lsst.ap.association.metrics.NumberNewDiaObjectsMetricTask
+    config:
+      connections.labelName: diaPipe  # partial name of metadata dataset
+  numUnassociatedDiaObjects:
+    class: lsst.ap.association.metrics.NumberUnassociatedDiaObjectsMetricTask
+    config:
+      connections.labelName: diaPipe
+  fracUpdatedDiaObjects:
+    class: lsst.ap.association.metrics.FractionUpdatedDiaObjectsMetricTask
+    config:
+      connections.labelName: diaPipe
+  numTotalSolarSystemObjects:
+    class: lsst.ap.association.metrics.NumberSolarSystemObjectsMetricTask
+    config:
+      connections.labelName: diaPipe
+  numAssociatedSsObjects:
+    class: lsst.ap.association.metrics.NumberAssociatedSolarSystemObjectsMetricTask
+    config:
+      connections.labelName: diaPipe
+  totalUnassociatedDiaObjects:
+    class: lsst.ap.association.metrics.TotalUnassociatedDiaObjectsMetricTask
+  numSciSources:
+    class: lsst.ip.diffim.metrics.NumberSciSourcesMetricTask
+    config:
+      connections.sources: initial_stars_footprints_detector
```

Comment:
Just chatted with @ebellm about this (he happened to be nearby): it's fine that `numSciSources` will have a different meaning now (and `fracDiaSourcesToSciSources` will be quite different, too, and potentially > 1!). We should either fix that description, or rename it to be what the description says. If we are going to report such a value, it would have to come from an afterburner-style detection and measurement catalog, a la [RFC-938](https://jira.lsstcorp.org/browse/RFC-938), which we don't have yet.

Reply:
I think I'd prefer renaming, just so that the same metric always means the same thing.

So what exactly are "initial stars", and what is the significance of their number?

Reply:
`initial_stars` is defined as `Catalog of unresolved sources detected on the calibrated exposure.` As currently designed, it's the list of isolated S/N>10 psf-like sources that are used for astrometric and photometric calibration. The significance of their number tells you something about how many good isolated sources there are on the science image, but I don't know if that's something we really care about as a metric. What was the purpose of `numSciSources` as it was implemented here?

If we rename this, `numSciSources` would be `None` until we get that afterburner implemented.

Reply:
> What was the purpose of `numSciSources` as it was implemented here?

@ebellm is probably the better source for that, but I assumed it's a measure of how crowded the field is. And presumably the number of DiaSources we expect is proportional to the total number of sources, not just the isolated ones.

> If we rename this, `numSciSources` would be `None` until we get that afterburner implemented.

Exactly what I had in mind.

Reply:
The very 🏃 meeting discussion seemed to agree that we don't particularly care about counting the initial stars.

I suppose that means all the "derived" metrics would also be removed? (We really should have had a standalone "number of DiaSources" metric and done the math in Chronograf, but we didn't appreciate that at the time, and it was before [DMTN-085](https://dmtn-085.lsst.io/) made that the policy.)

Reply:
Per AP meeting discussion today, I've removed all four of these "sci sources" metrics.

Yeah, we probably should have a "number of diaSources" metric. It may be worthwhile to have "number of calibration sources" as well (i.e. the output of this new task). I think that's work for another ticket, though.